spec/factories/mediated_pages.rb

```diff
@@ -31,11 +31,20 @@ def needed_date_is_valid; end
     association :user, factory: :sequence_webauth_user
   end
 
-  factory :hoover_archive_mediated_page, parent: :mediated_page do
-    origin { 'HV-ARCHIVE' }
-    origin_location { 'SOMEWHERE-30' }
-    destination { 'HV-ARCHIVE' }
+  factory :education_mediated_page, class: 'MediatedPage' do
+    item_id { '1234' }
+    origin { 'EDUCATION' }
```

Comment:
Education isn't a great library for our fixtures -- they'll probably (?)go back to regular paging once the building reopens 🤷‍♂️

Reply:
that's good to know - is Art an OK alternative? i just need somewhere that isn't using Aeon